ingest-app/src/cmr/ingest/services/jobs.clj

```diff
@@ -203,10 +208,21 @@
        context
        (ingest-events/provider-autocomplete-suggestion-reindexing-event provider)))))
 
+(defn trigger-bulk-granule-task-cleanup
+  [context]
+  (info "Sending events to cleanup bulk granule update tasks in all providers:")
+  (ingest-events/publish-gran-bulk-update-event
+   context
+   (ingest-events/granule-bulk-update-task-cleanup-event)))
+
 (def-stateful-job BulkUpdateStatusTableCleanup
   [_ system]
   (bulk-update-status-table-cleanup {:system system}))
 
+(def-stateful-job BulkGranUpdateTaskCleanup
+  [_ system]
+  (trigger-bulk-granule-task-cleanup {:system system}))
```

Comment:
This should just invoke the function to do the cleanup directly rather than going through the event.